### DISCUSSION TITLE
Audio waveforms are not visible in FCP

### DISCUSSION THREAD

**Discussion by markspen** [latenitefilms/BRAWToolbox#35] [Support] [answered]
When placing the sync clip with audio in the timeline, there is no audio waveform - the audio portion of the clip is black. I can hear the audio.

**Comment by latenitefilms** [comment, marked as answer]
That's strange. It's just a standard WAV file - we're not doing anything tricky or fancy with the audio.

**Reply by latenitefilms** [reply]
I'm going to mark this as the answer - as it must be a Final Cut Pro bug/glitch - as all we're doing is supplying a standard WAV file.
